desktop-environment.el

```diff
@@ -357,22 +361,36 @@ portion of the screen."
 ;;; Commands - screenshots
 
 ;;;###autoload
-(defun desktop-environment-screenshot ()
+(defun desktop-environment-screenshot (delay)
   "Take a screenshot of the screen in the current working directory."
-  (interactive)
+  (interactive "P")
   (let ((default-directory (expand-file-name desktop-environment-screenshot-directory)))
-    (start-process-shell-command "desktop-environment-screenshot" nil desktop-environment-screenshot-command)))
+    (if (and delay
+             (numberp delay)
+             (> delay 0))
+        (start-process-shell-command "desktop-environment-screenshot"
+                                     nil
+                                     (concat desktop-environment-screenshot-command
+                                             (format desktop-environment-screenshot-delay-argument delay)))
+        (start-process-shell-command "desktop-environment-screenshot" nil desktop-environment-screenshot-command))))
 
 ;;;###autoload
-(defun desktop-environment-screenshot-part ()
+(defun desktop-environment-screenshot-part (delay)
   "Take a partial screenshot in the current working directory.
```

Comment:
can you please add a reference to `DELAY` in the docstring as well as a reference to `desktop-environment-screenshot-delay-argument`?

Reply:
should be fixed in 73ef705

Reply:
Should be resolved